packages/cyberstorm/src/components/Layout/BaseLayout/BaseLayout.module.css

```diff
@@ -14,7 +14,7 @@
   --w: 56vw;
 
   position: absolute;
-  top: -3.5rem;
+  top: calc(var(--header-height) - var(--header-height) *2);
```

Comment:
Why not just `calc(-1 * var(--header-height))` ?

Reply:
That's advanced math